frontend/apps/reader/modules/readergesture.lua

```diff
@@ -339,9 +339,9 @@ function ReaderGesture:buildMenu(ges, default)
         {"toggle_gsensor", Device:canToggleGSensor()},
         {"toggle_rotation", not self.is_docless, true},
         {"toggle_reflow", not self.is_docless, true},
-        {"toggle_wifi", Device:hasWifiToggle() and "toggle_wifi", true},
-        {"wifi_on", Device:hasWifiToggle() and "wifi_on", true},
-        {"wifi_off", Device:hasWifiToggle() and "wifi_off", true},
+        {"toggle_wifi", Device:hasWifiToggle(), true},
+        {"wifi_on", Device:hasWifiToggle(), true},
+        {"wifi_off", Device:hasWifiToggle(), true},
```

Comment:
Still too many separators :)
All the 3 wifi ones should be in the same separated area.
And I guess you want to move toggle_reflow below them, near the Zoom to fit as it's kopt stuff

Reply:
Depends on whether you see it as the "toggle section" or the "zoom-type stuff section". I suppose "zoom stuff" makes more sense as a grouping.